utility/support.cpp

```diff
@@ -501,6 +501,33 @@ size_t fc_strlcpy(char *dest, const char *src, size_t n)
   }
 }
 
+size_t fc_strlcpy(char *dest, const QStringView &src, size_t n)
```

Comment:
It would make sense to call this function from the `const char *` overload above